RR_Main Analysis_Secondary_Analysis.R

```diff
@@ -0,0 +1,144 @@
+# Load necessary packages
+chooseCRANmirror()
+install.packages("coin_1.4-3-1.tgz", repos = NULL, type = "source")
+install.packages("survival_3.5-8.tgz", repos = NULL, type = "source")
+install.packages("libcoin_1.0-10.tgz", repos = NULL, type = "source")
+install.packages("readxl")
+install.packages("censReg")
+install.packages("maxLik")
+install.packages("miscTools")
+library(miscTools)
+library(libcoin)
+library(survival)
+library(dplyr)
+library(coin)
+library(readxl)
+library(dplyr)
+library(censReg)
+library(maxLik)
```

Comment:
The only 3 packages that are used here are `{readxl}`, `{dplyr}` and `{ggplot2}, if I am correct. If I'm correct, please adjust this part accordingly.